Orange/widgets/data/owsql.py

```diff
@@ -61,22 +62,28 @@ def __init__(self):
         box = gui.widgetBox(vbox)
         self.servertext = QtGui.QLineEdit(box)
         self.servertext.setPlaceholderText('Server')
+        self.servertext.setToolTip('server')
         if self.host:
             self.servertext.setText(self.host if not self.port else
                                     '{}:{}'.format(self.host, self.port))
         box.layout().addWidget(self.servertext)
         self.databasetext = QtGui.QLineEdit(box)
-        self.databasetext.setPlaceholderText('Database')
+        self.databasetext.setPlaceholderText('Database[/Schema]')
+        self.databasetext.setToolTip('database[/schema]')
```

Comment:
For someone who isn't used to square brackets optional notation, this tooltip isn't exactly helpful.